Sources/Hyperspace/HTTP/HTTP.swift

```diff
@@ -172,18 +175,20 @@ public struct HTTP {
         /// Initialize a new `Request` with any given `URL`, HTTP headers and body.
         /// - Parameters:
         ///   - url: The `URL` for which this request was created.
+        ///   - method: The HTTP method for this request
```

Comment:
Minor - please add a period here too.
```suggestion
        ///   - method: The HTTP method for this request.
```